### ISSUE TITLE
Update data structure

### ISSUE THREAD

**Issue by varmar05** [lutraconsulting/anaconda#12]
Some postgres tables are not used at all, some of them are not linked properly. 

For tsvis integration need to resolve:
- sensor vs sensors
- location vs catchments
- variables

**Comment by ICHydro** [comment]
Yes we know about they unused tables. Those are part of future plans (for instance adoption OGC standards). For now:

- sensor vs sensors: let's use "sensor" for now, but we may want to add a location_id which is also a foreign key to the locations table.

- location vs catchments: we should use location. Catchment is only relevant for one type of sensor (river gauge).

- variables: can you elaborate?
